options.go

```diff
@@ -134,6 +134,10 @@ type FileCopyMethod struct {
 	fcopy func(src, dest string, info os.FileInfo, opt Options) (err error, skipFile bool)
 }
 
+// The default FileCopyMethod.
+// This only is changed during tests.
+var defaultCopyMethod = CopyBytes
```

Comment:
> Users can set their opt.FileCopyMethod by theirselves, NOT by env or any other implicit mechanism.

The users of `copy` do not have any way to change this as a default. It can only be changed by developers of `copy`, and is only ever changed by the `all_test.go` code.

It's test-specific code, and I agree that's not ideal. My earlier comments explained why it's needed, though.

Maybe naming it `default` is misleading? I could change it to `unspecifiedFileCopyMethodOverride`

Reply:
I ended up removing this. I replaced it with a function that runs at the end of `getDefaultOptions` and replaces the `FileCopyMethod`.